types/mparticle__web-sdk/index.d.ts

```diff
@@ -3,6 +3,45 @@ import { Batch } from "@mparticle/event-models";
 // Placeholder for Dictionary-like Types
 export type Dictionary<V = any> = Record<string, V>;
 
+// Rokt Manager Types
+export interface RoktPartnerAttributes {
+    [key: string]: string | number | boolean | undefined | null;
+}
+
+export interface RoktPartnerExtensionData<T> {
+    [extensionName: string]: T;
+}
+
+export interface RoktSelectPlacementsOptions {
+    attributes: RoktPartnerAttributes;
+    identifier?: string;
+}
+
+export type RoktPlacement = Record<string, unknown>;
```

Comment:
I think the correct type is (here](https://github.com/ROKT/sdk-web/blob/main/packages/web-sdk/controller/src/controller/platform/services/placements/placement.ts#L31) but confirm with Matt B/Guillermo